setup.cfg

```
@@ -34,96 +34,93 @@ license_files =
 
 [options]
 packages = find_namespace:
-python_requires = >=3.8.1,<3.10
+python_requires = >=3.8.1,<3.11
 include_package_data = True
 install_requires =
-    aiohttp==3.7.4.post0
+    aiohttp==3.8.1
     aiohttp-json-rpc==0.13.3
-    aiosqlite==0.17.0
-    appdirs==1.4.4
+    aiosignal==1.2.0
     apsw-wheels==3.36.0.post1
-    async-timeout==3.0.1
-    attrs==21.2.0
+    async-timeout==4.0.2
+    attrs==21.4.0
     Babel==2.9.1
-    cffi==1.14.6
-    chardet==4.0.0
-    click==8.0.1
-    colorama==0.4.4
+    charset-normalizer==2.0.12
+    click==8.1.2
     commonmark==0.9.1
     contextlib2==21.6.0
     discord.py @ git+https://github.com/Rapptz/discord.py@3d914e08e0c7df370987592affb3655d2a12f7d1#egg=discord.py
-    distro==1.6.0; sys_platform == "linux"
-    fuzzywuzzy==0.18.0
-    idna==3.2
-    Markdown==3.3.4
-    multidict==5.1.0
-    psutil==5.8.0
-    pycparser==2.20
-    Pygments==2.10.0
-    PyNaCl==1.4.0
+    distro==1.7.0; sys_platform == "linux"
+    frozenlist==1.3.0
+    idna==3.3
+    importlib-metadata==4.11.3
+    jarowinkler==1.0.2
+    Markdown==3.3.6
+    multidict==6.0.2
+    orjson==3.6.7
+    platformdirs==2.5.1
+    psutil==5.9.0
+    Pygments==2.11.2
     python-dateutil==2.8.2
-    python-Levenshtein-wheels==0.13.2
-    pytz==2021.1
-    PyYAML==5.4.1
+    pytz==2022.1
+    PyYAML==6.0
+    rapidfuzz==2.0.8
     Red-Commons==1.0.0
     Red-Lavalink==0.11.0rc0
-    rich==10.9.0
-    schema==0.7.4
+    rich==12.2.0
```

```suggestion
    rich==12.4.1
```

Not too sure about this one, but I have it running locally with 12.4.1 and had no issues.